tests/migration_test.go

```diff
@@ -1048,13 +1049,20 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				cfgMap, err = virtClient.CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Get(virtconfig.ConfigMapName, options)
 				Expect(err).ToNot(HaveOccurred())
 				originalMigrationConfig = cfgMap.Data["migrations"]
-				tests.UpdateClusterConfigValueAndWait("migrations", `{"progressTimeout" : 5, "completionTimeoutPerGiB": 5}`)
+
+				data := map[string]string{
+					"progressTimeout":         "5",
+					"completionTimeoutPerGiB": "5",
+					"bandwidthPerMigration":   "1Mi",
+				}
+				migrationData, err := json.Marshal(data)
+				Expect(err).ToNot(HaveOccurred())
+				tests.UpdateClusterConfigValueAndWait("migrations", string(migrationData))
 			})
 			AfterEach(func() {
 				tests.UpdateClusterConfigValueAndWait("migrations", originalMigrationConfig)
 			})
-			PIt("[test_id:2227] [flaky] should abort a vmi migration without progress", func() {
-				tests.SkipStressTestIfRunnigOnKindInfra()
+			PIt("[test_id:2227] should abort a vmi migration on timeout", func() {
```

Comment:
Not sure the new name is better. I mean yes we are waiting for a timeout, but the timeout is not a migration-lifetime timeout. The timeout is related to the seeing progress on the migration.

Reply:
we still have PIt ?

Reply:
I am fine dropping the name change (I asked about it when posting it).
Please confirm.

Reply:
Yes. As described above we should enable it with #3542, since even before we had the invalid-config issue it was very flaky. With the issue which @EdDev is fixing it will go back to that state. So let's fix the flake before re-enabling it.

Reply:
> I am fine dropping the name change (I asked about it when posting it).
> Please confirm.

Since you have to rebase anyway, please. It is slightly less confusing.

Reply:
ack